app/models/User.scala

```diff
@@ -37,28 +39,29 @@ class UserTableDef(tag: Tag) extends Table[User](tag, "user") {
     (id, firstName, lastName, mobile, email) <>(User.tupled, User.unapply)
 }
 
-object Users {
+@Singleton
+class Users @Inject() (protected val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[JdbcProfile] {
 
-  val dbConfig = DatabaseConfigProvider.get[JdbcProfile](Play.current)
+  import driver.api._
```

Comment:
This import can only be done inside of the class, as driver is an inherited member of HasDatabaseConfigProvider